src/elements/content-explorer/ItemGridCell.js

```diff
@@ -26,26 +24,20 @@ const ItemGridCell = ({
 }: Props) => {
     return (
         <figure className="bce-ItemGridCell">
-            <ItemGridThumbnail item={item} />
-            <figcaption>
+            <ItemGridThumbnail item={item} onItemClick={onItemClick} />
```

Comment:
Seems like this would only allow you to click on the thumbnail? Would we instead want to click the whole grid item to select?